src/app/content/highlights/components/EditCard.tsx

```diff
@@ -44,6 +44,7 @@ export interface EditCardProps {
   data?: HighlightData;
   className: string;
   shouldFocusCard: boolean;
+  role?: string;
```

Comment:
This doesn't seem to be used?

Reply:
You are correct. That got leftover from trying to figure out where role needed to be specified.